frontend/public/locales/fr/apply-child.json

```
@@ -274,15 +274,15 @@
     "yes": "Oui",
     "no": "Non",
     "exit": "Sortie",
-    "close-application": "(FR) Close application",
+    "close-application": "Fermer la demande",
     "exit-link": "https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires.html",
     "status-checker-link": "https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires/demande.html",
     "modal": {
-      "header": "(FR) Close application",
-      "info": "(FR) By closing the application, you will end your session and any information entered will be cleared from your browser. Make sure you printed or save a copy of your application code.",
-      "are-you-sure": "Are you sure you want to close the application?",
-      "back-btn": "(FR) Back",
-      "close-btn": "(FR) Close application"
+      "header": "Fermer la demande",
+      "info": "En fermant cette demande, vous mettrez fin à votre session et les informations saisies seront effacées de votre navigateur. Assurez-vous d'avoir une copie de votre code de demande.",
+      "are-you-sure": "Êtes-vous sûr de vouloir fermer la demande\u00a0?",
```

No space before a question mark `?`

```suggestion
      "are-you-sure": "Êtes-vous sûr de vouloir fermer la demande?",
```

---

Removed the space